inst/include/Rcpp/Reference.h

```diff
@@ -59,15 +59,15 @@ namespace Rcpp{
         Reference_Impl( const std::string& klass ) {
             SEXP newSym = Rf_install("new");
             Shield<SEXP> call( Rf_lang2( newSym, Rf_mkString( klass.c_str() ) ) );
-            Storage::set__( Rcpp_eval( call ) ) ;
+            Storage::set__( Rcpp_eval( call , Rcpp::internal::get_Rcpp_namespace()) );
```

Comment:
@kevinushey How about if I merge that and run another rev.dep. check just in case?